### ISSUE TITLE
When view:cache, Unable to locate a class or view for component [pages.default]

### ISSUE THREAD

**Issue by onemoreahmad** [tanthammar/tall-forms#122]
**Describe the bug**
Got an error when run `php artisan view:cache`

**Screenshots**
<img width="861" alt="Screen Shot 2023-01-11 at 10 16 19 AM" src="https://user-images.githubusercontent.com/1885785/211742364-15a60c3c-9541-4908-b89c-b7ea48a4bc2c.png">
<img width="672" alt="Screen Shot 2023-01-11 at 10 17 49 AM" src="https://user-images.githubusercontent.com/1885785/211742373-b7cfd159-12fa-43b9-ae4d-e48114347e63.png">

**Environment:**
Latest 

 - $wrapWithLayout true/false: both, tried with true and false 
It doesn't matter if I use my own **wrappWithLayout** or not, both works just fine but I got the error only on terminal when I run view:cache command 

Thanks.

**Comment by tanthammar** [comment]
Did you check your setting for the layout path?
Also the error msg says it's looking for the component for "pages.default" if it is any clue to you.
(unable to reproduce on a fresh installation)

**Comment by onemoreahmad** [comment]
@tanthammar, Yes I did check, but has nothing to do with the error. 

I think page.default component something (maybe) from extra packages you support, like jetstream or laravel breeze ? I don't know because I don't use neither of them and if you use them and one of them may has this "pages.default" component. because I search for default component in your package but found nothing.

**Comment by tanthammar** [comment]
I guess you read this page?
https://tina-hammar.gitbook.io/tall-forms/installation/manual-installation/wrapper-view

What is the content of your "Wrapper view"?

This package default view assumes that you use Jetstream or Breeze, which installs views/components/pages/default.blade.php

The content of this package wrapper

```blade
<x-pages.default :title="$form->formTitle">
    <div class="px-6 sm:px-0">
        @include('tall-forms::form')
    </div>
</x-pages.default>
```

Does this help your case?

**Comment by onemoreahmad** [comment]
Yes, and I've created this template and I'm using it and it worked just fine. The issue I'm having is I can't view:cache from terminal any more because of that page.default issue.

Let me ask you to clarify things, why are you using `<x-pages.default>`  while there is no pages/default blade file there in the whole package? 
Because if I just commented the content of that file, it work just fine. 
<img width="729" alt="Screen Shot 2023-01-11 at 7 10 10 PM" src="https://user-images.githubusercontent.com/1885785/211857288-e5f3938c-46be-4e67-8f70-62c442c02d7a.png">

and thanks for your time 💌
